tests/unit/scrapers/test_xiaohongshu.py

```
@@ -1292,7 +1293,7 @@ async def test_not_xhs_raises(self):
             mock_client.__aexit__ = AsyncMock(return_value=None)
             MockClient.return_value = mock_client
 
-            with pytest.raises(RuntimeError, match="did not redirect to xiaohongshu.com"):
+            with pytest.raises(ScraperError, match="did not redirect to xiaohongshu.com"):
```

Ruff (0.15.7)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)